### PR TITLE
refactor(login): remove feature and ui libraries

### PR DESCRIPTION
**Pull request by RemiBonnet**
# What does this PR do?

Remove Feature and UI libraries

---

### Global

- [ ] This PR does not introduce any breaking change
- [ ] This PR introduces breaking change(s) and has been labeled as such
- [ ] I have found someone to review this PR and pinged him

### Store

- [ ] This PR introduces new store changes

### NX

- [ ] I have run the dep-graph locally and made sure the tree was clean i.e no circular dependencies
- [ ] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`

### Clean Code

- [ ] I made sure the code is type safe (no any)
- [ ] I have included a feature flag on my feature, if applicable

### CODE REVIEW

File: apps/console/src/app/app.tsx [hunk 1]
```diff
@@ -140,7 +140,7 @@ export function App() {
   return (
     <GTMProvider state={gtmParams}>
       <Routes>
-        <Route path={LOGIN_URL} element={<LoginPage />} />
+        <Route path={LOGIN_URL} element={<PageLogin />} />
```
Comment (bdebon) [inline apps/console/src/app/app.tsx:143]: Do you want that we apply this convention for all the project => `PageName` ?

File: libs/pages/login/src/lib/feature/container/container.tsx [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
 import { AuthEnum, useAuth } from '@console/shared/auth'
 import { useOrganization } from '@console/domains/organization'
 import { useProjects } from '@console/domains/projects'
+import LayoutLogin from '../../ui/layout-login/layout-login'
+import Login from '../../ui/login/login'
 
 // import posthog from 'posthog-js'
 
-export function LoginPage() {
+export function Container() {
```
Comment (bdebon) [inline libs/pages/login/src/lib/feature/container/container.tsx:13]: Why container and not `PageLoginFeature`? For each Page we gonna have its Container if I understand well?

File: apps/console/src/app/app.tsx (hunk 1 above)
Comment (RemiBonnet) [reply apps/console/src/app/app.tsx:143]: Yes I think to automatically generate this we can rename the folder "Pages" to "Page" ?

File: libs/pages/login/src/lib/feature/container/container.tsx (hunk 2 above)
Comment (RemiBonnet) [reply libs/pages/login/src/lib/feature/container/container.tsx:13]: I agree, better to have `PageLoginFeature` I rename this one

File: apps/console/src/app/app.tsx (hunk 1 above)
Comment (bdebon) [reply apps/console/src/app/app.tsx:143]: Ah cool, schematics will then create our components with the name PageBlabla? That's a good trick! I like it! Do you want to make the renaming?

**Pull request merged**